scripts/ci.requirements.txt

```diff
@@ -7,9 +7,13 @@
 # 2.4.4 is the version in Ubuntu 20.04. It supports Python >=3.5.
 pylint == 2.4.4
 
-# Use the last version of mypy that is compatible with Python 3.6.
-# Newer versions should be ok too.
-mypy >= 0.971
+# Use a version of mypy that is compatible with our code base.
+# mypy <0.940 is known not to work: see commit
+#  :/Upgrade mypy to the last version supporting Python 3.6
```

Comment:
Is `:/` intentional? If so, what does it mean?

Reply:
It's a Git revision.
```
git show -s ':/Upgrade mypy to the last version supporting Python 3.6'
```

Reply:
Oh great, thanks!